### ISSUE TITLE
InvalidClientTaaAcceptanceError time too precise error if container timezone is not UTC

### ISSUE THREAD

**Issue by Ennovate-com** [hyperledger/aries-cloudagent-python#2419]
When running in a container where the timezone is not UTC, acceptance of the TAA results in an error:

``` indy_vdr.error.VdrError: Request failed: client request invalid:
 InvalidClientTaaAcceptanceError('Txn Author Agreement acceptance time 1691989200 is too precise and is a privacy risk.',)```

The agreement acceptance time should always reflect a time of midnight in UTC, regardless of the default timezone.

**Comment by Ennovate-com** [comment]
This change should be made wherever the TAA acceptance time is being calculated. There are two places in ACA-Py fixed by PR #2420 places and one place in Indy VDR fixed by PR #211.

**Comment by Ennovate-com** [comment]
Fixed by PR #2420
